docs/fundamentals/toc.yml

```diff
@@ -432,8 +432,12 @@ items:
     href: ../core/deploying/runtime-patch-selection.md
   - name: Single file deployment and executable
     href: ../core/deploying/single-file.md
-  - name: Trim self-contained deployments and executables
-    href: ../core/deploying/trim-self-contained.md
+  - name: Trim self-contained deployments
+    items:
+    - name: Overview and how-to
```

Comment:
@adegeo Is this okay with you? Here's the preview: https://review.docs.microsoft.com/en-us/dotnet/core/deploying/trim-self-contained?branch=pr-en-us-20671.